### PR TITLE
11-5 Email

### PR DESCRIPTION
**Pull request by EUdds**

### CODE REVIEW

**Comment by jmvera255** [comment]
I think we are avoiding making the office hours link public? @lmichael107 ?

**Comment by lmichael107** [comment]
Yes, please make sure it's scrubbed from web postings (and anywhere else
that's public) and replace with something like 'See emails from CHTC staff
or login to a CHTC server for links to the office hours virtual room."

File: user-news.md [hunk 1]
```diff
@@ -12,6 +12,27 @@ For older updates not shown on this page, see our [user mailing list
...
+
+-   We are currently draining jobs from all remaining nodes in the old cluster configuration. **Jobs requesting 7 days (or any time request that cannot be completed by Nov 10) will not start.** Jobs can still run on the old cluster nodes, but only if their time request ("--time=" in the submit file) indicates that they will complete before next Tuesday, November 10. 
+-   **If you have not already logged into the new cluster configuration and tried to run test jobs, please do so** as soon as possible! After next week, the old cluster will not be available to run jobs -- you are less likely to experience interruptions to your work if you make sure everything is running smoothly on the new cluster now.
+
+In addition, we are hoping to have user modules for the Intel compilers and libraries within the next few weeks, and we will send a separate email when these are available.
+
+Please send any questions to <chtc@cs.wisc.edu>, as always, and join us at office hours (Tu/Th 3-5pm,[  https://go.wisc.edu/chtc-officehours](https://go.wisc.edu/chtc-officehours)).
```
Comment (ChristinaLK) [inline user-news.md:30]:
```suggestion
Please send any questions to <chtc@cs.wisc.edu>, as always, and join us at office hours (Tu/Th 3-5pm, see the CHTC server login message for the meeting link).
```

File: user-news.md [hunk 2]
```diff
@@ -12,6 +12,27 @@ For older updates not shown on this page, see our [user mailing list
...
+
+-   We are currently draining jobs from all remaining nodes in the old cluster configuration. **Jobs requesting 7 days (or any time request that cannot be completed by Nov 10) will not start.** Jobs can still run on the old cluster nodes, but only if their time request ("--time=" in the submit file) indicates that they will complete before next Tuesday, November 10. 
+-   **If you have not already logged into the new cluster configuration and tried to run test jobs, please do so** as soon as possible! After next week, the old cluster will not be available to run jobs -- you are less likely to experience interruptions to your work if you make sure everything is running smoothly on the new cluster now.
+
+In addition, we are hoping to have user modules for the Intel compilers and libraries within the next few weeks, and we will send a separate email when these are available.
+
+Please send any questions to <chtc@cs.wisc.edu>, as always, and join us at office hours (Tu/Th 3-5pm, see the CHTC server login message for the meeting link).
```
Comment (ChristinaLK) [inline user-news.md:30]:
```suggestion
Please send any questions to <chtc@cs.wisc.edu>, as always, or join us at office hours (Tu/Th 3-5pm, see emails from CHTC staff or login to a CHTC server for links to the office hours virtual room).
```

**Review by ChristinaLK — APPROVED** [review]

File: user-news.md [hunk 3]
```diff
@@ -12,6 +12,27 @@ For older updates not shown on this page, see our [user mailing list
 archives](https://www-auth.cs.wisc.edu/lists/chtc-users/).
 
 
+
+------------------------------------------------------------------------
+## Limited Job Starts on Previous HPC Clusuter
```
Comment (ChristinaLK) [inline user-news.md:17]:
```suggestion
## Limited Job Starts on Previous HPC Cluster
```

File: user-news.md [hunk 4]
```diff
@@ -12,6 +12,27 @@ For older updates not shown on this page, see our [user mailing list
...
 
+
+------------------------------------------------------------------------
+## Limited Job Starts on Previous HPC Cluster
+### Thursday, November 5, 2020
+Greetings CHTC users, 
+
+*The below pertains to users of CHTC's HPC Cluster, for which users submit jobs via the [aci-service-1.chtc.wisc.edu](http://aci-service-1.chtc.wisc.edu) login node. Users of only the HTC System are welcome to ignore the below.*
+
+As [previously announced](https://chtc.cs.wisc.edu/user-news.shtml)**, **next week we will be transitioning all remaining hardware from the previous HPC cluster configuration to the new configuration. This means:
```
Comment (ChristinaLK) [inline user-news.md:23]:
```suggestion
As [previously announced](https://chtc.cs.wisc.edu/user-news.shtml), next week we will be transitioning all remaining hardware from the previous HPC cluster configuration to the new configuration. This means: 
```

**Comment by EUdds** [comment]
Whoops missed this thread. Thanks @ChristinaLK

**Pull request merged**